root/usr/share/nethesis/NethServer/Module/Dashboard/SystemStatus/Backup.php

```diff
@@ -48,7 +48,45 @@ private function readBackup()
         $backup['vfs'] = $br['VFSType'] ? $br['VFSType'] : '-';
         $backup['status'] = $br['status'];
         $backup['time'] = $br['BackupTime'];
-
+        
+        $orario = explode(" ",$backup['time']);
+      
+        function formatOre($h){
```

Comment:
Move functions in the outer scope and set them as private.
It's a matter of code readability.

Reply:
If I move functions out of private readBackup() function, they will not work. Also if I rename them in "private" in the same function...

Reply:
Yes you can access them, you just need to use the right scope.
Example:
```php
private function formatHour($x)
{
    if(strlen($h)==1){return "0".$h;}else{return $h;}
}

private function readBackup()
{
    ...
    $this->formatHour($h)
    ...
}
```

By the way, you can format the date using just one line of code, and replace the entire function like this:
```php
private function readBackup()
{
    ...
   $backup['time'] = $view->translate('Every day at ').sprintf('%02d', $hour[1]) .":".sprintf('%02d', $hour[0]);
   ...
}
```

Also take a look to your `formatOre` and `formatMin`: the two function are identical! ;)